servers/publikator/index.js

```diff
@@ -1,3 +1,19 @@
 require('@orbiting/backend-modules-env').config()
 const server = require('./server')
-server.run()
+const throng = require('throng')
+
+const {
+  CLUSTER,
+  WEB_CONCURRENCY = 1
+} = process.env
+
+if (CLUSTER) {
+  throng({
+    workers: WEB_CONCURRENCY,
+    grace: 30000,
+    start: server.run,
```

Comment:
`start`, `master` receive a worker ID as an argument. Might be useful for debuging purposes. If possible, even have listeners (on `SIGTERM`) able to log worker ID.

[A Complex example](https://github.com/hunterloftis/throng#a-complex-example) looks pretty much like that.